ledger/src/blockstore_db.rs

```diff
@@ -1014,7 +1168,53 @@ impl<'a> WriteBatch<'a> {
     }
 }
 
-fn get_cf_options(access_type: &AccessType) -> Options {
+struct PurgedSlotFilter<C: Column + ColumnName>(Slot, CString, PhantomData<C>);
```

Comment:
nit: Could we convert this into a struct with named fields for clarity?

Reply:
done! https://github.com/solana-labs/solana/pull/16697/commits/d3aebd4b08b50df757c38a2007726af0fa4e334e https://github.com/solana-labs/solana/pull/16697/commits/9925bf26835f2a887e2c49d6531d4030a1fe2219